charts/hono/example/add_example_data_device_registry.sh

```diff
@@ -11,7 +11,11 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #*******************************************************************************
+{{- if ( eq .Values.deviceRegistryExample.hono.registry.http.insecurePortEnabled true ) }}
 HTTP_BASE_URL="http://{{ include "hono.fullname" . }}-service-device-registry:8080/v1"
+{{- else }}
+HTTP_BASE_URL="https://{{ include "hono.fullname" . }}-service-device-registry:8443/v1"
+{{- end }}
```

Comment:
Minor:
The helm template directive causes a bash syntax error to be displayed in the editor (preventing also shellcheck checks of the rest of the script).
This could be avoided by using something like:
```bash
URL_SCHEME="{{- if ( eq .Values.deviceRegistryExample.hono.registry.http.insecurePortEnabled true ) }}http{{ else }}https{{ end }}"
URL_PORT=$([ "$URL_SCHEME" = "http" ] && echo "8080" || echo "8443")
HTTP_BASE_URL="${URL_SCHEME}://{{ include "hono.fullname" . }}-service-device-registry:${URL_PORT}/v1"
```